vscode-dotnet-runtime-library/src/Acquisition/WinMacGlobalInstaller.ts

```diff
@@ -181,34 +182,57 @@ We cannot verify .NET is safe to download at this time. Please try again later.`
         return installerPath;
     }
 
-    private async installerFileHasValidIntegrity(installerFile : string) : Promise<boolean>
+    private async userChoosesToContinueWithInvalidHash() : Promise<boolean>
     {
-        const realFileHash = await this.file.getFileHash(installerFile);
-        this.acquisitionContext.eventStream.post(new DotnetFileIntegrityCheckEvent(`The hash of the installer file we downloaded is ${realFileHash}`));
-        const expectedFileHash = this.installerHash;
-        this.acquisitionContext.eventStream.post(new DotnetFileIntegrityCheckEvent(`The valid and expected hash of the installer file is ${expectedFileHash}`));
+        const yes = validationPromptConstants.allowOption
+        const no = validationPromptConstants.cancelOption;
```

Comment:
nit: I know semicolons are not required, but maybe it should be in the .editorconfig/tslint.json to suggest them for consistency? Here, the `yes` line doesn't have one.

Reply:
Good catch! unfortunately I couldnt find an editorconfig setting, it looks like there might be an eslint one. First, we would need to change from tslint to eslint, so I think that should be done separately